rfcs/card-component.md

```diff
@@ -0,0 +1,56 @@
+# Card Component
+
+## Exports
+
+* `Card`
+* `StyledAction`
+* `StyledBody`
+* `StyledContents`
+* `StyledHeaderImage`
+* `StyledRoot`
+* `StyledThumbnail`
+* `StyledTitle`
+
+## `Card` API
+
+* `action: React.Node` - Optional
+  Contents of the `StyledAction` component
+* `children: React.Node` - Optional
+  Contents of the `StyledBody` component
+* `hasThumbnail: ({+thumbnail?: string}) => boolean` - Optional
+  Function that takes `Card` props and returns a boolean that represents if a thumbnail will be rendered
+* `headerImage: string` - Optional
+  `src` of the `StyledHeaderImage` component
+* `overrides: {Action, Body, Contents, HeaderImage, Root, Thumbnail, Title}` - Optional
+  * `Action: OverrideObjectT` - Optional
+  * `Body: OverrideObjectT` - Optional
+  * `Contents: OverrideObjectT` - Optional
+  * `HeaderImage: OverrideObjectT` - Optional
+  * `Root: OverrideObjectT` - Optional
+  * `Thumbnail: OverrideObjectT` - Optional
+  * `Title: OverrideObjectT` - Optional
+    Overrides for presentational components.
+* `thumbnail: string` - Optional
+  `src` of the `StyledThumbnail` component
+* `title: React.Node` - Optional
+  Contents of the `StyledTitle` component
+
+## Usage
```

Comment:
can we move the usage to be the first section to keep consistency with other RFCs?